### PR TITLE
Make source work for data with missing modules

### PR DESCRIPTION
**Pull request by maxnoe**
Missing pixels are marked as `*_failing_pixels`, and the waveforms are filled with invalid data markers:

* Fill uint16 maxval  (2¹⁶ - 1) in r0
* Fill nan in r1

### CODE REVIEW

**Comment by maxnoe** [comment]
I will now try to create a small test data file from one of the data files from last night

**Review by moralejo — APPROVED** [review]
Seems ok to me, if no surprises show up in the tests.

**Comment by maxnoe** [comment]
I uploaded the test data file with 50 events from run 3369

**Review by FrancaCassol — APPROVED** [review]

File: ctapipe_io_lst/_version.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+version = '0.6.0'
```
Comment (rlopezcoto) [inline ctapipe_io_lst/_version.py:4]: why is this version `0.6.0`?

Comment (maxnoe) [reply ctapipe_io_lst/_version.py:4]: Dam, accidental commit. File should not have been added.

**Pull request merged**